contentcuration/contentcuration/frontend/channelEdit/vuex/contentNode/index.js

```diff
@@ -29,13 +29,26 @@ export default {
        *
        * E.g. Reading -> Math -> Integrals -> Physics
        * would be represented here as
-       * [
-       *   ['id-reading', 'id-math'],
-       *   ['id-math', 'id-integrals'],
-       *   ['id-integrals', 'id-physics']
-       * ]
+       * {
+       *   'id-reading': ['id-math'],
```

Comment:
Sorry, I will update the code comment to be accurate. There can't be nested levels, I am just using an object to do quick lookup by key rather than storing in an array that needs to be searched.